pavelib/utils/test/bokchoy_utils.py

```diff
@@ -77,6 +77,9 @@ def wait_for_server(server, port):
         )
     )
 
+    if tasks.environment.dry_run:
```

Comment:
What's the use case for dry-run? Should we add info about it to the testing.rst doc?